src/python/pants/util/osutil.py

```diff
@@ -18,7 +20,7 @@
 Pid = int
 
 
-def get_os_name(uname_result=None):
+def get_os_name(uname_result: Optional[posix.uname_result] = None) -> str:
```

Comment:
Things like `posix.uname_result` are what I want types for :)

Reply:
Yes! So much clearer. Btw, I found this via the typeshed repository, the entry for `os.uname()`. They have fantastic types for the stdlib (and some 3rd parties).

--

Another trick: if you use IntelliJ or Pycharm, hold down the command key and hover over any call to `get_os_name` :)